Initializer/Initialize.cs

```diff
@@ -14,23 +14,62 @@ public class Initialize
         private ManageAccessToEntity _manageAccessToEntity = new ManageAccessToEntity();
 
         private readonly string _pathRoot = Path.GetFullPath(Path.Combine(
-            Directory.GetCurrentDirectory(), @"..\..\..\..\LanguageSkills"));
+            Directory.GetCurrentDirectory(), @"..\..\..\..\LanguageSkills\wwwroot"));
         private bool _isData = true;
         private string _tempItem = "";
         private int _countStage = 0;
+        private List<Language> _allLanguages = new List<Language>();
 
-        private string _createPath(string directoryName, string fileName, string fileExtension)
+        private string _generatePath(string directoryName, string fileName, string fileExtension)
         {
-            return @"\wwwroot\Dictionary\" + directoryName + fileName + fileExtension;
+            //Generate a global path
+            string path = @"\Dictionary\" + directoryName + fileName + fileExtension;
+            return _isFileExist(path) ? path : 
+                _removeSpaces(@"\Dictionary\" + directoryName, fileName + fileExtension);
+        }
+
+        private bool _isFileExist(string path)
+        {
+            return File.Exists(_pathRoot + path);
+        }
+
+        private string _removeSpaces(string path, string fileName)
+        {
+            if (_isFileExist(path + fileName.Replace(" ", "")))
+            {
+                return path + fileName.Replace(" ", "");
+            }
+            else
+            {
+                Console.WriteLine("File doesn't exist " + _pathRoot + path + fileName);
+                return null;
+            }
         }
 
         public ExcelWorksheet GetDataFromFile(string path)
         {
-            FileInfo fileInfo = new FileInfo(path);
-            ExcelPackage.LicenseContext = LicenseContext.NonCommercial;
-            ExcelPackage package = new ExcelPackage(fileInfo);
-            ExcelWorksheet worksheet = package.Workbook.Worksheets.FirstOrDefault();
-            return worksheet;
+            if (path != null)
+            {
+                try
+                {
+                    FileInfo fileInfo = new FileInfo(path);
+                    ExcelPackage.LicenseContext = LicenseContext.NonCommercial;
+                    ExcelPackage package = new ExcelPackage(fileInfo);
+                    ExcelWorksheet worksheet = package.Workbook.Worksheets.FirstOrDefault();
+                    return worksheet;
+                }
+                catch (Exception e)
+                {
+                    Console.WriteLine(e);
```

Comment:
Bad approach for writing error - it's does not return useful information.
Instead you can write something like "{ClassName} - {MethodName}. Exception message: {e.Message}. Exception stack trace: {e.StackTrace}".

Please 1) try this approach, 2) see how much useful the output information looks for you, 3) compare with your initial approach, 4) fix in all files in your solution.